src/components/App.jsx

```diff
@@ -43,23 +43,15 @@ const router = createHashRouter([
 const settings = getWeb3Settings()
 
 const { chains, provider } = configureChains(
-  [mainnet, polygon, bsc],
+  [polygon],
```

Comment:
I am not completely sure it's a good idea to remove other chains from here as the provider for mainnet is used to fetch proposal for example.

Reply:
This was requested.